karton/core/main.py

```diff
@@ -1,11 +1,125 @@
 import argparse
+import logging
+import os.path
+from configparser import ConfigParser
 from typing import Any, Dict, List
 
+from minio import Minio
+from redis import StrictRedis
+
 from .__version__ import __version__
 from .backend import KartonBackend
 from .config import Config
 from .karton import Consumer
 
+log = logging.getLogger(__name__)
+
+
+def get_user_option(prompt: str, default: str) -> str:
+    user_input = input(prompt + f"\n[{default}] ")
```

Comment:
Why not just put everything in f-string:
```suggestion
    user_input = input(f"{prompt}\n[{default}] ")
```